### ISSUE TITLE
Pressing enter on an input field of a option settings does not save it

### ISSUE THREAD

**Issue by elyezer** [ansible/awx#3022]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
Pressing enter on an input field when editing settings does not save the settings but flips the first switch option on the page. For example editing any input on the system settings and then pressing enter will update the `Enable Administrator Alerts` switch instead of saving the options (expected behavior for any form on a web UI)

##### ENVIRONMENT
* AWX version: devel
* AWX install method: docker
* Ansible version:  X.Y.Z
* Operating System: Fedora 29 (Linux)
* Web Browser: Chrome

##### STEPS TO REPRODUCE

1. Navigate to Settings > System (or any other settings page
2. Click on any input field and press enter
3. Watch the first switch  (on/off) on the page and it will be updated

##### EXPECTED RESULTS

The page or the options that is being edited would be saved

##### ACTUAL RESULTS

Only the switch is being updated, not the option being edited.

**Comment by mabashian** [comment]
For usability purposes I definitely agree that hitting enter while focused on a form should submit the form.  That being said, I don't know that this has ever worked in the awx UI.  I think that we should definitely fix the inadvertent toggle and then have a conversation about what we expect the `enter` key to do moving forward.

**Comment by elyezer** [comment]
@mabashian I find myself pressing enter whenever I change an input field and it is annoying that anything happens.

**Comment by ryanpetrello** [comment]
@elyezer to your knowledge, is this still an issue in `devel`?

**Comment by elyezer** [comment]
@ryanpetrello yes, basically pressing enter does not work in any input field, even when trying to log in.
